src/payment/components/ProductLineItem.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ProductLineItem extends React.PureComponent {
+  renderSeatType(seatType) {
+    const selectedSeatType = (() => {
+      switch (seatType) {
+        case 'professional':
+        case 'no-id-professional':
+          return 'Professional Certificate';
+        case 'verified-certificate':
+          return 'Verified Certificate';
+        case 'honor':
+        case 'audit':
+        default:
+          return null;
+      }
+    })();
+    return (
+      <p>
+        {selectedSeatType}
+      </p>
+    );
+  }
+
+  render() {
+    const {
+      product,
+    } = this.props;
+    return (
+      <div className="pb-3">
+        <h2 className="h6">In your cart</h2>
+        <p>Your purchase contains the following:</p>
```

Comment:
i18n

Reply:
Do I need a description in this case in the `<FormattedMessage ... />`? If yes, would it be something like "Cart heading"?